samples/Rewrite/ConditionParser/ConditionParserContext.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Rewrite.ConditionParser
+{
+    public class ConditionParserContext
```

Comment:
This could be called `InputReader` or something like that because it's used to read a string char by char